glue_jupyter/common/state_widgets/viewer_histogram.py

```diff
@@ -2,19 +2,30 @@
 import traitlets
 from ...state_traitlets_helpers import GlueState
 from ...vuetify_helpers import link_glue_choices
+from glue.viewers.
+
+from glue.viewers.histogram.viewer import MatplotlibHistogramMixin
 
 __all__ = ['HistogramViewerStateWidget']
 
 
-class HistogramViewerStateWidget(v.VuetifyTemplate):
+class HistogramViewerStateWidget(v.VuetifyTemplate, MatplotlibHistogramMixin):
     template_file = (__file__, 'viewer_histogram.vue')
     x_att_items = traitlets.List().tag(sync=True)
     x_att_selected = traitlets.Int(allow_none=True).tag(sync=True)
     glue_state = GlueState().tag(sync=True)
 
     def __init__(self, viewer_state):
         super().__init__()
+        MatplotlibHistogramMixin.setup_callback(self)
 
         self.glue_state = viewer_state
 
         link_glue_choices(self, viewer_state, 'x_att')
+
+    def vue_axis_to_bins(self):
+        # self.glue_state.
+        MatplotlibHistogramMixin._update_axes(self)
```

Comment:
You shouldn't need to inherit from or use this mixin - there should be a method on the histogram viewer state class for syncing the bins to the current limits.